.eslintrc

```diff
@@ -50,29 +70,51 @@
         "commonjs": true,
         "allowThis": true,
         "exceptions": [
-          { "property": "propTypes" },
-          { "property": "defaultProps" },
-          { "property": "current" }
+          {
+            "property": "propTypes"
+          },
+          {
+            "property": "defaultProps"
+          },
+          {
+            "property": "current"
+          }
+        ]
+      }
+    ],
+    "no-console": [
+      "warn",
+      {
+        "allow": [
+          "warn",
+          "error"
         ]
       }
     ],
-    "no-console": ["warn", { "allow": ["warn", "error"] }],
-    "no-unused-vars": ["warn", {
-      "argsIgnorePattern": "^props$",
-      "ignoreRestSiblings": true
-    }],
+    "no-unused-vars": [
+      "warn",
+      {
+        "argsIgnorePattern": "^props$",
+        "ignoreRestSiblings": true
+      }
+    ],
     "lines-between-class-members": "off",
     "max-classes-per-file": "off",
     "spaced-comment": "warn"
   },
   "settings": {
     "import/resolver": {
       "node": {
-        "moduleDirectory": ["node_modules", "./src"],
-        "extensions": [".js", ".jsx", ".ts", ".tsx"]
-      },
-      "import/parsers": {
-        "@typescript-eslint/parser": [".ts", ".tsx"]
```

Comment:
Removing this extra setting that is breaking the linter on Windows

Reply:
Actually, the indentation in the file was wrong and these had to be under settings instead of being under "import/resolver"